### PR TITLE
Fixed bug in create_mar.sh script of Text_to_Speech_Synthesizer

### PR DESCRIPTION
**Pull request by AshwinChafale**
## Description

Fixed a bug in create_mar.sh script of "text_to_speech_synthesizer" model.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: examples/text_to_speech_synthesizer/create_mar.sh [hunk 1]
```diff
@@ -2,13 +2,28 @@
 set -euxo pipefail
 
 cd /tmp
-rm torchhub.zip
+if [ -f "torchhub.zip" ] 
+then
+   rm torchhub.zip
+fi
 wget https://github.com/nvidia/DeepLearningExamples/archive/torchhub.zip
```
Comment (maaquib) [inline examples/text_to_speech_synthesizer/create_mar.sh:9]: Why not just use `-O` to overwrite the file?
```suggestion
wget -O https://github.com/nvidia/DeepLearningExamples/archive/torchhub.zip torchhub.zip
```

**Review by maaquib — CHANGES_REQUESTED** [review]
Inline comment

File: examples/text_to_speech_synthesizer/create_mar.sh [hunk 2]
```diff
@@ -2,13 +2,26 @@
 set -euxo pipefail
 
 cd /tmp
-rm torchhub.zip
-wget https://github.com/nvidia/DeepLearningExamples/archive/torchhub.zip
-rm -rf DeepLearningExamples-torchhub
+
+wget https://github.com/nvidia/DeepLearningExamples/archive/torchhub.zip -O torchhub.zip
+
+if [ -d "DeepLearningExamples-torchhub" ]
```
Comment (maaquib) [inline examples/text_to_speech_synthesizer/create_mar.sh:8]: Use wget with -O instead of all the if-else statements

**Review by maaquib — CHANGES_REQUESTED** [review]

File: examples/text_to_speech_synthesizer/create_mar.sh [hunk 3]
```diff
@@ -2,13 +2,26 @@
...
+   rm -rf DeepLearningExamples-torchhub
+fi
 unzip torchhub.zip
+
 cd -
-rm tacotron.zip
-rm -rf PyTorch
+if [ -f "tacotron.zip" ] 
+then
+   rm tacotron.zip
```
Comment (maaquib) [inline examples/text_to_speech_synthesizer/create_mar.sh:17]: Doesn't `rm -rf tacotron.zip` suffice? What's the reason for using an `if`

**Review by maaquib — CHANGES_REQUESTED** [review]

**Review by chauhang — APPROVED** [review]

**Review by maaquib — APPROVED** [review]

**Pull request merged**